doc/deploy.md

```diff
@@ -0,0 +1,69 @@
+# Deploy Chaos Mesh
+
+This document describes how to deploy Chaos Mesh for performing chaos experiments 
+on your application on Kubernetes. 
+
+## Prerequisites
+
+Before deploying Chaos Mesh, make sure the following items have been installed:
+
+- Kubernetes >= v1.12
+- [RBAC](https://kubernetes.io/docs/admin/authorization/rbac) enabled (optional)
+- [Helm](https://helm.sh/) version >= v2.8.2
+
+## Step 1: Get Chaos Mesh
+
+```bash
+git clone https://github.com/pingcap/chaos-mesh.git
+cd chaos-mesh/
+```
+
+## Step 2: Create custom resource type
+
+To use Chaos Mesh, you must first create the related custom resource type.
+
+```bash
+kubectl apply -f manifests/crd.yaml
+```
+
+## Step 3: Install Chaos Mesh
+
```

Comment:
Depending on your environment, there are following methods of installing Chaos Mesh: